ibexa/commerce/5.0/encore/webpack.config.js

```diff
@@ -1,11 +1,22 @@
-const Encore = require('@symfony/webpack-encore');
 const path = require('path');
-const getIbexaConfig = require('./ibexa.webpack.config.js');
-const ibexaConfig = getIbexaConfig(Encore);
-const customConfigs = require('./ibexa.webpack.custom.configs.js');
-const {
-    isReactBlockPathCreated,
-} = require('./ibexa.webpack.config.react.blocks.js');
+const fs = require('fs');
+
+const Encore = require('@symfony/webpack-encore');
```

Comment:
I wonder whether this common part could also be extracted? 🤔

Reply:
importing Encore is needed here nevertheless, as this file works on it. In terms of lines 8-11 - these could be extracted to frontend-config module, but there is one caveat - inside that module we would need to do sth like this `const bundles = require(path.resolve('./var/encore/ibexa.config.js'));` as `require` uses relative to file instead of cwd. 
I think it's better to sacrifice these few lines instead of doing some strange requires with resolves inside.